pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -143,6 +146,34 @@ func (m *managerImpl) IsUnderInodePressure() bool {
 	return hasNodeCondition(m.nodeConditions, api.NodeInodePressure)
 }
 
+func startMemoryThresholdNotifier(thresholds []Threshold, observations signalObservations, hard bool, handler func()) error {
+	for _, threshold := range thresholds {
+		if threshold.Signal != SignalMemoryAvailable ||
+			(hard && threshold.GracePeriod != time.Duration(0)) ||
+			(!hard && threshold.GracePeriod == time.Duration(0)) {
+			continue
+		}
+		observed, found := observations[SignalMemoryAvailable]
+		if !found {
+			continue
+		}
+		// TODO: remove hardcoded root memory cgroup path
```

Comment:
use: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cm/helpers_linux.go#L26 to find the mount point.